src/app/services/jit_transcoder.py

```
@@ -658,6 +658,29 @@ async def get_transcoding_info(
+    async def find_existing_windows(
+        self, asset_hash: str, profile_hash: str
+    ) -> list[int]:
+        """
+        查找给定资产和配置的所有现有窗口ID
+
+        Args:
+            asset_hash: 资产哈希
+            profile_hash: 配置哈希
+
+        Returns:
+            list[int]: 现有窗口ID列表，按ID排序
```

Ruff (0.13.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)